.github/workflows/reusable-build-docs.yaml

```diff
@@ -69,7 +69,9 @@ jobs:
 
     - name: Build the Ansible community package docs
       run: >-
-        make webdocs ANSIBLE_VERSION="${{
+        make webdocs ${{
+          inputs.generate-redirects && 'EXTRA_TAGS="-t redirects"' || ''
+        }} ANSIBLE_VERSION="${{
```

Comment:
@webknjaz Hey again, thanks for the previous comment that made me re-think this. What I had here before was garbage. I'm not sure how it got muddled up so my apologies.

When the `generate-redirects` input is true, the command is `make webdocs EXTRA_TAGS="-t redirects" ANSIBLE_VERSION=""`

When `generate-redirects` is false, the command is `make webdocs  ANSIBLE_VERSION=""`. It does look like there's an extra space between `webdocs` and `ANSIBLE_VERSION` but I don't think that has any effect in how the make command gets evaluated.